src/ui.lua

```diff
@@ -218,8 +218,12 @@ end
 
 function LFGAnnouncementsUI:_createInstanceContainer(instanceId)
 	local instances = Instances
-	local name = instances:GetInstanceName(instanceId)
-
+	local instanceName = instances:GetInstanceName(instanceId)
+	local instanceLevelRange = instances:GetLevelRange(instanceId)
+	local name = instanceName
+	if getn(instanceLevelRange) == 2 then
+	    name = stringformat("%s (%d, %d)", instanceName, instanceLevelRange[1], instanceLevelRange[2])
```

Comment:
This will alter the actual name of the container. We don't want that. This should be done in `getGroupTitle`